x-pack/plugins/uptime/server/graphql/monitors/schema.gql.ts

```diff
@@ -65,10 +71,33 @@ export const monitorsSchema = gql`
     y: Int
   }
 
+  "Represents a monitor's duration performance in microseconds at a point in time."
+  type MonitorDurationAreaPoint {
+    "The timeseries value for this point in time."
+    x: UnsignedInteger!
+    "The min duration value in microseconds at this time."
+    y0: Float
```

Comment:
I realllly dislike the `y0`, `y` distinction, but I'm OK with it here since this is a private API.

If we ever make it public I'd like it to be `yMin`,`yMax` or similar. I'll leave it to you whether you feel it should be changed for this PR.

Reply:
Oh ok - I don't have any objection to changing the name. I don't like that convention either.

Reply:
@andrewvc if you take a look at https://github.com/elastic/kibana/pull/30561/commits/e4ea5b5acc49945521bbc45922437b372577000c, let me know what you think.